### ISSUE TITLE
ExecutionFailed: Target Morsel cannot hold the data to copy

### ISSUE THREAD

**Issue by CelineWuest** [neo4j/neo4j#13336]
When running the following query:
```cypher
WITH * LIMIT 1
CALL {
	WITH null AS a0, [ a1 IN collect(0) ] AS a2, null AS a3
	RETURN 0 AS a4
}
MERGE ()
```

I encountered the following error:
```
Neo4jError: Neo.DatabaseError.Statement.ExecutionFailed (A bug has occurred in the pipelined runtime: The target morsel cannot hold the data to copy.)
```

I believe the query mentioned above is semantically and syntactically correct and thus no error should be thrown here.

I encountered this issue when testing queries against the **Neo4j 5.13.0 enterprise version** in a Docker container running **alpine v.3**. 

### Steps to reproduce
Run the following queries and observe it throws an error:
```cypher
WITH * LIMIT 1 CALL { WITH null AS a0, [ a1 IN collect(0) ] AS a2, null AS a3 RETURN 0 AS a4 } MERGE ()
```

### Expected behavior
The query should run successfully

### Actual behavior
The query fails with the error message `Neo4jError: Neo.DatabaseError.Statement.ExecutionFailed (A bug has occurred in the pipelined runtime: The target morsel cannot hold the data to copy.)`

**Comment by PetrJanouch** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [[GitHub issue]ExecutionFailed: Target Morsel cannot hold the data to copy](https://trello.com/c/knTxXYI8/5205-github-issueexecutionfailed-target-morsel-cannot-hold-the-data-to-copy)

**Comment by PetrJanouch** [comment]
Hi @DominicWuest ,
thanks for reporting this. This is a rare occasion when the code itself admits this is a bug 😄

**Comment by sherfert** [comment]
We merged a fix that will appear in 5.15.
